.github/workflows/deploy-frontends-to-production.yml

```diff
@@ -146,8 +104,8 @@ jobs:
   ### deploy website ###
   website:
     name: build-push-deploy-website
-    needs: [check, image-tag]
-    if: needs.check.outputs.run_website == 'true'
+    needs: [image-tag]
+    if: inputs.website == true
```

Comment:
If the selection for app is true, then run job

Reply:
Thanks @123MwanjeMike ....at what point are these apps selected? Is the selection automated?

Reply:
Hi @Baalmart. The selection is at the point of trigger, as describe/illustrated in the PR description.

Reply:
Thanks @123MwanjeMike for the clarification notes. Just have a few more inquiries:
1. Could you share more details about this manual selection?
2. How does this manual selection change/improve current status? What is our main objective?

Reply:
Hi @Baalmart, in accordance to the inquiries;
1. It may be a little difficult for one to appreciate this PR until they manually test it as described in the PR description.
2. This PR addresses card [PLATOPS-28](https://airqoteam.atlassian.net/browse/PLATOPS-28?atlOrigin=eyJpIjoiZGU5OWEzM2JkZDExNGY5MThjYmMwNWQwZWE0OTcwYjkiLCJwIjoiaiJ9) on the Jira board.
![image](https://user-images.githubusercontent.com/41787587/197772512-8b299a42-c0ed-4c16-8e71-d78a23fe4073.png)